HelpSource/Reference/Server-Command-Reference.schelp

```diff
@@ -566,6 +566,7 @@ table::
 ## strong::int:: || buffer number
 ## strong::int:: || number of frames
 ## strong::int:: || number of channels (optional. default = 1 channel)
+## strong::float:: || the required sample rate (optional. default = the server's sample rate)
```

Comment:
After completion message! See comment in `Buffer.allocMsg`

Reply:
Side note: we would also need an explanation that `0.0 has the special meaning of "use the server's sample rate".

(Since `channels` and `completionMsg` are both optional, `sampleRate` would need to stay optional, too.)